libs/agno/agno/os/app.py

```diff
@@ -1015,11 +1017,20 @@ def serve(
             )
         )
 
+        # Adding *.yaml to reload_includes to reload the app when the yaml config file changes.
+        if reload:
+            if reload_includes is not None:
```

Comment:
What if the user doesn't want this? They'd have to explicitly set it in the excludes? I think we should only set it if `reload_includes` is not set at all